apps/prairielearn/src/pages/studentInstanceQuestion/studentInstanceQuestion.js

```diff
@@ -280,10 +283,34 @@ router.get(
 router.get(
   '/',
   asyncHandler(async (req, res) => {
-    const variant_id =
+    let variant_id =
       res.locals.assessment.type === 'Exam' || typeof req.query.variant_id !== 'string'
         ? null
         : req.query.variant_id;
+
+    const isAssessmentAvailable =
+      _.get(res.locals, 'assessment_instance.open', true) &&
+      _.get(res.locals, 'authz_result.active', true);
```

Comment:
My understanding is that these variables are always set at this point, so this can be simpler:

```suggestion
    const isAssessmentAvailable =
      res.locals,assessment_instance.open &&
      res.locals.authz_result.active;
```

Reply:
Can confirm, they would be set in selectAndAuthzInstanceQuestion.